web/src/components/Footer.tsx

```diff
@@ -13,6 +13,7 @@ const FooterContainer = styled.footer`
   justify-content: center;
   padding: 15px 5px;
   background-color: ${props => props.theme.colors.backgroundAccentColor};
+  box-shadow: 0 2px 3px 3px rgba(0, 0, 0, 0.1);
```

Comment:
added box shadow to look similar to the `Header` box shadow and to separate from the PoiList Toolbar

Reply:
Looking at the designs the footer should not look like this on the maps page, but I guess it is not part of this ticket.

Reply:
yes we have to decide what happens with the footer, maybe it gonna be a small overlay on the map view. But yes separate ticket